### PR TITLE
docs: Satellite sleep behaviour

### PR DESCRIPTION
**Pull request by brandonSc**
Probably don't want to merge until the sleep feature is rolled out completely.

### CODE REVIEW

File: docs/cloud/satellites/managing.md [hunk 1]
```diff
@@ -115,6 +115,26 @@ earthly sat launch <satellite-name>
 
 The newly launched satellite will always get the latest version available.
 
+### Managing a satellite's state
+
+To save costs, satellites automatically enter a "sleep" state after 30 min of inactivity. While a satellite is asleep, you are not billed for any compute minutes.
```
Comment (vladaionescu) [inline docs/cloud/satellites/managing.md:120]:
```suggestion
To save costs, satellites automatically enter a **sleep** state after 30 min of inactivity. While a satellite is asleep, you are not billed for any compute minutes.
```

File: docs/cloud/satellites/managing.md [hunk 2]
```diff
@@ -115,6 +115,26 @@ earthly sat launch <satellite-name>
...
+
+The satellite will automatically "wake up" when a new build is started while it's in a sleep state. This is visible during the `Init` phase of the Earthly log.
```
Comment (vladaionescu) [inline docs/cloud/satellites/managing.md:122]:
```suggestion
The satellite will automatically **wake up** when a new build is started while it's in a sleep state. This is visible during the `Init` phase of the Earthly log.
```

File: docs/cloud/satellites/managing.md [hunk 3]
```diff
@@ -115,6 +115,26 @@ earthly sat launch <satellite-name>
...
+earthly sat sleep <satellite-name>
+```
+
+Similarly, a Satellite can be manually woken up using:
+
+```bash
+earthly sat wake <satellite-name>
+```
+
+Note that the [`inspect`](#Checking status of a satellite) command will show you if a Satellite is currently awake or asleep.
```
Comment (vladaionescu) [inline docs/cloud/satellites/managing.md:136]: I don't think this link works - you might need to check the preview to see what the correct anchor should be here.

**Review by vladaionescu — COMMENTED** [review]
:+1: Some small remarks...

File: docs/cloud/satellites/managing.md (hunk 3 above)
Comment (brandonSc) [reply docs/cloud/satellites/managing.md]: I think I have the syntax right now, but it doesn't seem like it's creating the new preview link?

Comment (vladaionescu) [reply docs/cloud/satellites/managing.md]: Not sure why - maybe it needs yet another commit?

**Review by vladaionescu — APPROVED** [review]

File: docs/cloud/satellites/managing.md (hunk 3 above)
Comment (brandonSc) [reply docs/cloud/satellites/managing.md]: It looks right now btw, triggered it with some dummy commits

**Pull request merged**